flutter/shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -45,23 +45,19 @@ const std::vector<std::string> kBindableSystemKeys = {
 // (ui/events/x/events_x_utils.cc).
 constexpr int32_t kScrollOffsetMultiplier = 53;
 
-double ComputePixelRatio(flutter::TizenViewBase* view) {
+double ComputePixelRatio(int32_t screen_dpi) {
```

Comment:
I'm not sure if it makes sense to maintain ComputePixelRatio() here since it's no longer a local function. Could it move to system_utils.h?